api/v1/errorHandler.js

```diff
@@ -54,6 +55,17 @@ function isDbError(err) {
 }
 
 /**
+ * Indicates whether the error is a Cache error as defined by our cacheErrors
+ * module.
+ *
+ * @param {Error} err - The error to test
+ * @returns {Boolean} true if err is defined as one of our Cache errors
+ */
+function isCacheError(err) {
+  return err instanceof cacheErrors.UpdateDeleteForbidden;
```

Comment:
Ideally ever error should be defined. Not sure why it was not defined when the redisError.js files were created.